pkgs/applications/graphics/qvge/default.nix

```
@@ -12,13 +12,14 @@
 
 stdenv.mkDerivation rec {
   pname = "qvge";
-  version = "0.6.3";
+  version = "0.6.3-untagged-2024-01-08";
```

```suggestion
  version = "0.6.3-unstable-2024-01-08";
```
See https://github.com/NixOS/nixpkgs/tree/master/pkgs#versioning